baseband/dada/header.py

```diff
@@ -336,15 +337,16 @@ def sample_shape(self, sample_shape):
         self['NPOL'], self['NCHAN'] = sample_shape
 
     @property
-    def bandwidth(self):
-        """Bandwidth covered by the data."""
-        return abs(self['BW']) * u.MHz
-
-    @bandwidth.setter
-    def bandwidth(self, bw):
-        bw = bw.to(u.MHz).value
+    def sample_rate(self):
+        """Number of complete samples per second."""
+        return (1. / self['TSAMP']) * u.MHz
+
+    @sample_rate.setter
+    def sample_rate(self, sample_rate):
+        sample_rate = sample_rate.to_value(u.MHz)
+        self['TSAMP'] = 1. / sample_rate
```

Comment:
I think this should be `abs(sample_rate)` since as written one can still pass in a negative one to indicate sideband. I think this should be noted in the docstring...